README.md

```diff
@@ -92,7 +92,7 @@
 
 - [Live Demo Link](https://tashbudgetapp.onrender.com)
```

Comment:
- [ ] Your project is working as intended :ok_hand: however, app appears not properly deployed. It throws an error in an when the signup button is clicked. Let's ensure that database and other configurations are correctly installed and that the deployed app runs as intended.

[renderError.webm](https://github.com/danielmatama/budget-app-rails/assets/12745474/d7abae06-574f-4efa-ad60-5f2f03cc1693)